src/table/virtualized-table/Body.tsx

```diff
@@ -144,6 +157,13 @@ const Body = forwardRef<BodyRef, BodyProps>((props, ref) => {
     [innerForwardRef, bodyHeight, rowMeta, rowCount, resizeVisibleCells, rowsInPage]
   );
 
+  const scrollLeft = layout.snapshotData ? viewService.scrollLeft : 0;
+  const scrollTopRatio = layout.snapshotData ? viewService.scrollTopRatio || 0 : 0;
+
+  useEffect(() => {
+    interpolatedScrollTo(scrollTopRatio, scrollLeft);
+  }, [scrollTopRatio, scrollLeft]);
```

Comment:
This is similar to https://github.com/qlik-oss/sn-table/blob/main/src/table/pagination-table/components/TableWrapper.tsx#L84

Reply:
@haxxmaxx `scrollTopRatio, scrollLeft` are not state variables. So changing them won't trigger a re-render thankfully. But on initial/first render it will tell `react-window` to scroll to a position, that might trigger a internal re-render of the `react-window` components. I do not think that is an issue, either the value is `0` and nothing should happen or the value is greater than `0` and it should scroll to the correct position in the snapshot. Given that all states are correct.

However not including `interpolatedScrollTo` in the dependency array might cause bugs. So I recommend finding a solution that allows you to include it. (wrap `interpolatedScrollTo` in a `useCallback` might to it..)